packages/icons/src/HarnessIcons.ts

```diff
@@ -339,6 +339,8 @@ import GitopsUnknown from './gitops-unknown.svg'
 import GitopsYes from './gitops-yes.svg'
 import GoogleKubernetesEngine from './google-kubernetes-engine.svg'
 import Google from './google.svg'
+import GovernancePolicyset from './governance-policyset.svg'
```

Comment:
Is "Policyset" considered one word for the platform or should it be separated?

```suggestion
import GovernancePolicySet from './governance-policy-set.svg'
```

Reply:
Good shout. Updated.